src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -85,6 +86,34 @@ Napi::Value CoreWrap::read_model_async(const Napi::CallbackInfo& info) {
     }
 }
 
+Napi::Value CoreWrap::read_model_from_buffer(const Napi::CallbackInfo& info) {
+    if (info.Length() < 1 || info.Length() > 2 || !info[0].IsBuffer()) {
+        Napi::TypeError::New(env, "Invalid arguments").ThrowAsJavaScriptException();
+        return info.Env().Undefined();
+    }
+
+    Napi::Buffer<uint8_t> model_data = info[0].As<Napi::Buffer<uint8_t>>();
```

Comment:
Do we always get here Napi::Buffer<uint8_t>? Is there a possibility that the user will pass a buffer with different type of data?